src/main.js

```diff
@@ -183,52 +197,48 @@ var manifestor = function(options) {
     if (!arguments.length) return _canvasImageStates;
     _canvasImageStates = state;
 
-    // if (!initial) {
-    //     jQuery.publish('annotationsTabStateUpdated' + this.windowId, this.tabState);
-    // }
-
     return _canvasImageStates;
   }
 
-  // function detailTransition(detailLayout) {
-  //     renderLayout(detailLayout);
-  // }
-  // function overviewTransition(selection) {
-  //     renderLayout(detailLayout);
-  // }
   function setScrollElementEvents() {
     var animationTiming = 1200;
     var interactionOverlay = d3.select(overlays[0]);
     if (canvasState().perspective === 'detail') {
       interactionOverlay
         .style('opacity', 0)
-        .transition()
-        .duration(animationTiming)
         .style('pointer-events', 'none');
 
       d3.select(scrollContainer[0])
-        .transition()
-        .duration(animationTiming)
         .style('pointer-events', 'none')
-        .style('overflow-x', 'hidden')
         .style('overflow-y', 'hidden');
-    } else if(!_zooming) {
 
+    } else if(!_zooming) {
       interactionOverlay
-        .style('opacity', 1)
+        .style('pointer-events', 'all')
         .transition()
-        .duration(animationTiming)
-        .style('pointer-events', 'all');
+        .duration(animationTiming/2)
+        .style('opacity', 1);
 
       d3.select(scrollContainer[0])
-        .transition()
-        .duration(animationTiming)
         .style('pointer-events', 'all')
-        .style('overflow-x', 'hidden')
         .style('overflow-y', 'scroll');
     }
   }
 
+  function disableZoomAndPan() {
```

Comment:
ZoomPerClick, etc. properties are multiplication factors, so "1" means "do not zoom". The defaults are re-added afterward.